WTWSMS/common/scripted_triggers/ACR_scripted_triggers.txt

```diff
@@ -83,6 +88,7 @@ is_ancient_religion_trigger = {
 		religion = hellenic_pagan
 		religion = egyptian_pagan
 		religion = aten_pagan
+		religion = assyrian_pagan
```

Comment:
That's an unneccessary duplicate. We need not create a new one when this one works perfectly.